akbc2013/python/superuser-init.py

```diff
@@ -0,0 +1,193 @@
+        akbc2013 = Group(CONFERENCE,
+            readers     = ['everyone'],
+            writers     = [CONFERENCE],
+            signatures  = ['OpenReview.net'],
+            signatories = [CONFERENCE],
+            members     = [],
+            web         = '../webfield/akbc2013_webfield.html')
+        groups.append(akbc2013)
+
+
+    if overwrite_allowed(COCHAIRS):
+        Program_Chairs = Group(COCHAIRS,
+            readers     = [CONFERENCE, COCHAIRS, SPC, PC],
+            writers     = [CONFERENCE],
+            signatures  = [CONFERENCE],
+            signatories = [COCHAIRS],
+            members     = [])
+        groups.append(Program_Chairs)
+
+
+    if overwrite_allowed(SPC):
+        spc = Group(SPC,
+            readers     = [CONFERENCE, COCHAIRS, SPC, PC],
+            writers     = [CONFERENCE], #the conference needs to be a writer whenever the process functions need to modify the group
+            signatures  = [CONFERENCE],
+            signatories = [SPC],
+            members     = [])
+        groups.append(spc)
+
+    if overwrite_allowed(PC):
+        pc = Group(PC,
+            readers     = [CONFERENCE, COCHAIRS, SPC, PC],
+            writers     = [CONFERENCE],
+            signatures  = [CONFERENCE],
+            signatories = [PC],
+            members     = [])
+        groups.append(pc)
+
+    ## Post the groups
+    for g in groups:
+        print "Posting group: ",g.id
+        openreview.post_group(g)
+
+    openreview.post_group(openreview.get_group('host'),CONFERENCE)
+    openreview.add_members_to_group(openreview.get_group('host'),CONFERENCE)
+
+
+
+    #########################
+    ##  SETUP INVITATIONS  ##
+    #########################
+
+
+    ## Create the submission invitation
+    reply = {
+        'forum': None,
+        'replyto': None,
+        'readers': {
+            'description': 'The users who will be allowed to read the above content.',
+            'values': ['everyone']
+        },
+        'signatures': {
+            'description': 'How your identity will be displayed with the above content.',
+            'values-regex': '~.*'
+        },
+        'writers': {
+            'values-regex': '~.*'
+        },
+        'content': {
+            'title': {
+                'description': 'Title of paper.',
+                'order': 1,
+                'value-regex': '.{1,250}',
+                'required':True
+            },
+            'authors': {
+                'description': 'Comma separated list of author names, as they appear in the paper.',
+                'order': 2,
+                'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
+                'required':True
+            },
+            'authorids': {
+                'description': 'Comma separated list of author email addresses, in the same order as above.',
+                'order': 3,
+                'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
+                'required':True
+            },
+            'TL;DR': {
```

Comment:
if this field doesn't appear in the papers we should remove it from the invitation